<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<head>
  <title>AlignQC requirements</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<link rel="stylesheet" type="text/css" href="styles/flex_textfont_sans.css" media="all" />
<!-- SANDSTONE AND TILE TEMPLATE (FLEXIBLE-WIDTH, SANS-SERIF TEXT FONT VERSION) PROVIDED BY STANFORD UNIVERSITY INFORMATION TECHNOLOGY SERVICES FOR NON-DEPARTMENTAL OR UNOFFICIAL USE BY STANFORD FACULTY, STAFF AND STUDENTS. DESIGNED IN 2009 BY DAVE REAM, IT SERVICES. FOR POLICIES AND GUIDELINES ON ACCESSIBILITY, COPYRIGHT, AND ON THE USE OF STANFORD'S NAME AND TRADEMARKS, INCLUDING THE UNIVERSITY SEAL, SEE http://soap.stanford.edu, http://www.stanford.edu/group/identity, and http://adminguide.stanford.edu --> 
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-50750315-1', 'uiowa.edu');
  ga('send', 'pageview');
</script>
</head>

<body>

<!--BEGIN TOP RED BAR -->
<div id="top_navbar">
  <!-- OPTIONAL TOP NAVIGATION OR TEXT AREA INSIDE RED BAR; BAR EXPANDS HEIGHT AUTOMATICALLY TO ACCOMMODATE CONTENT -->
   <ul>
    <li>AlignQC IS DEVELOPED BY <a href="http://www.augroup.org/">KIN FAI AU</a> with help from collaborators</li>
  </ul>
  <!-- END OF OPTIONAL TOP RED BAR CONTENT -->
</div>
<!-- END TOP RED BAR -->


 <div id="content_container">
  <!-- PAGE TITLE AREA AND BOX -->
    <div id="page_title_area">
    <!-- STANDARD PAGE TITLE; NO DECORATIVE GRAPHIC -->
    <!-- <div id="page_title_leftcontent"> -->
    <div id="page_title_leftcontent">
      <h1>AlignQC</h1>
      <ul>
        <li>AlignQC is a tool for <b>assessing an alignment, <br></b>and <b>generating reports</b> that are easy to share

      </ul>
    </div>
    
    <div id="page_title_rightcontent">
	<ul>


	    <li>&nbsp;</li>

		<li><img src="images/errorpattern.png" width="150" >&nbsp;&nbsp;&nbsp;&nbsp;</li>
		
	</ul>
    </div>

<p class="page_title_area_lower">Latest News:  AlignQC is released <a href="alignqc.html#news">read more</a></p>
    
    
  </div>
  <!-- END PAGE TITLE AREA AND BOX -->
  <div id="content">
    <h2>Requirements</h2>
    AlignQC aims to provide useful quality control information based on an Alignment file. If any of the
    following requirements conflict with your lab's set up, please <a href="AlignQC_contact.html">contact</a> us. 
    
    <h3><img src="images/dna.png"> System Requirements</h3>
    <ul>
      <li>Linux operating system (Linux's coreutils must be 8.6 or better)</li>
    </ul>
    
    <h3><img src="images/dna.png"> Software Requirements</h3>
    <ul>
      <li>Python 2.7+</li>
      <li>R</li>
    </ul>
    
    <h3><img src="images/dna.png">Alignment file</h3>
    <ul>
      <li><b>BAM alignment - </b>If you analyze a transcriptome we recommend aligning with GMAP using the 'samse' format option and using samtools to convert this output to a BAM format.  The ideal format outputs multiple alignment paths for each read, and includes the query sequence.</li>
    </ul>
    <h3><img src="images/dna.png">Reference files</h3>
    <p>Only the alignment in BAM formant is necessary but to acheive all the analyses you should specify reference fasta and a reference genepred transcriptome</p>
    <ul>
      <li><b>--reference - </b>genome FASTA file (optional, but must be included to have error pattern analysis)</li>
      <li><b>--annotation - </b>refFlat genePred file.  See our <a href="http://www.augroup.org/IDP/IDP_gpd_format.asp">detailed description of the genePred file format</a>.  This is the main primary set of refernece transcripts used define known transcripts, and to annotate. (optional, but must be included to use annotations and genomic features in the report) </li>
    </ul>
    
  </div>
  
  
<div id="sidebar">
    <h2>Site Map</h2>
    <ul>
      <li><a href="alignqc">Home</a></li>
      
      <li><a href="AlignQC_download_current.html">Downloads</a></li>
      <li><a href="AlignQC_contact.html">Contact</a></li>
      <li><a href="AlignQC_requirements.html">Requirements</a></li>
      
  
    </ul>
    <h3>Help</h3>
    <ul>
      
     

      <li><a href="AlignQC_manual.html">Manual</a></li>
      
      
    </ul>
    <h2>Download</h2>
    <h3>Current release</h3>
    <ul>
      <li><a href="AlignQC_download_current.html"><img src="images/download.png"> AlignQC 1.2</a> (<a href="AlignQC_download_current.html#notes">notes</a>)</li>
    </ul>

    <h2>Useful links</h2>
    <ul>
      <li><a href="http://www.augroup.org">Au Lab</a> - Au Lab homepage </li>
    </ul>
    <ul>
      <li><a href="http://www.stanford.edu/group/wonglab/SpliceMap/">SpliceMap</a> - Novel splice junction discovery</li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/LSC/LSC">LSC</a> - Error correction of PacBio long reads </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP/IDP">IDP</a> - Isoform Detection and Prediction  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP-ASE/IDP-ASE">IDP-ASE</a> - Determine haplotypes and quantify allele specific expression  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP-fusion/IDP-fusion">IDP-Fusion</a> - Isoform Detection and Prediction across fusion genes  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP/hESC.html">hESC transcriptome</a> - hESC transcriptome by IDP </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/AlignQC/alignqc">AlignQC</a> - Check an alignment, assess error patterns, and make reports </li>
    </ul>
	
<ul>
      <li><a href="http://www.augroup.org/IDP-denovo.20160829/idp-denovo">IDP-denovo</a> - <i>de novo</i> transcriptome assembly and annotation</li>
    </ul>


<ul>
      <li><a href="http://www.augroup.org/GenoClone/GenoClone/GenoClone">GenoClone</a> - Reveal tumor heterogeneity</li>
    </ul>

 <ul>
      <li><a href="https://github.com/jason-weirather/Au-public">Au lab scripts</a> - Public repository of Au lab scripts and utilities </li>
    </ul>

    <br><ul><center><img src="images/seal.jpg" width="75%" hight="75%"><center></ul>
  </div>
<div id="footer">
    <ul>
      <li>&copy; <a href=" http://www.augroup.org/">Kin Fai Au</a>.  All Rights Reserved.</li>
      <li><a href="https://www.osu.edu/">The Ohio State University</a>, OH 43210.</li>
    </ul>
</div>

  
<!-- end #content -->
</body>


</html>
