<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>The Statistical Bioinformatics Lab @ The Ohio State University </title>

<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<link rel="stylesheet" type="text/css" href="styles/flex_textfont_sans.css" media="all" />
<!-- SANDSTONE AND TILE TEMPLATE (FLEXIBLE-WIDTH, SANS-SERIF TEXT FONT VERSION) PROVIDED BY STANFORD UNIVERSITY INFORMATION TECHNOLOGY SERVICES FOR NON-DEPARTMENTAL OR UNOFFICIAL USE BY STANFORD FACULTY, STAFF AND STUDENTS. DESIGNED IN 2009 BY DAVE REAM, IT SERVICES. FOR POLICIES AND GUIDELINES ON ACCESSIBILITY, COPYRIGHT, AND ON THE USE OF STANFORD'S NAME AND TRADEMARKS, INCLUDING THE UNIVERSITY SEAL, SEE http://soap.stanford.edu, http://www.stanford.edu/group/identity, and http://adminguide.stanford.edu --> 
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-50750315-1', 'uiowa.edu');
  ga('send', 'pageview');

</script>
</head>


<body>

<!--BEGIN TOP RED BAR -->
<div id="top_navbar">
  <!-- OPTIONAL TOP NAVIGATION OR TEXT AREA INSIDE RED BAR; BAR EXPANDS HEIGHT AUTOMATICALLY TO ACCOMMODATE CONTENT -->
   <ul>
    <li><a href="http://www.augroup.org/">AU LAB - THE STATISTICAL BIOINFORMATICS LAB @ The Ohio State University</a></li>
  </ul>
  <!-- END OF OPTIONAL TOP RED BAR CONTENT -->
</div>
<!-- END TOP RED BAR -->

<div id="content_container">
  <!-- PAGE TITLE AREA AND BOX -->
   <div id="page_title_area">
    <!-- STANDARD PAGE TITLE; NO DECORATIVE GRAPHIC -->
    <!-- <div id="page_title_leftcontent"> -->
    <div id="page_title_leftcontent">
      <h1>Au Lab</h1>
      <ul>
    </div>


  </div>


  <!-- PAGE TITLE LEFT CONTENT WITH OPTIONAL QUAD LAMP DECORATION; UN-COMMENT TO USE -->


  <!-- END PAGE TITLE AREA AND BOX -->
  <div id="content">


</p>


<h2>Openings</h2> </p>
<ul>The Au Lab is seeking candidates for two postdoctoral research scholar positions to work on sequencing analyses:</ul>
    <ul>
	<li><b>1) Postdoctoral Research Scholar in bioinformatics method development:</b></li>
		<ul> We are looking for a postdoctoral scholar to develop statistical and computational/algorithmic methods for high-throughput sequencing data analyses, such as transcriptome and epigenetics using PacBio and Oxford Nanopore Technologies. Background on mathematics, statistics, computer science or degree in a related field is desirable.</ul>
		<li><b>2) Postdoctoral Research Scholar in computational biology</b></li>
		<ul>We are looking for a postdoctoral scholar in biology background. Our lab performs world leading science in development process and cancer research. Our team focuses on making meaningful advances to understand development process and cancer, using cutting edge technologies (third generation sequencing technology) to create authentic models and gain novel biological insight in transcriptome and epigenetics level. A typical candidate would have skills in including but not limited to implementing basic in-house bioinformatics tools, collaborating with statistician and computational biologists to elucidate biological discovery and mining out biological insights. Background on biology, computational biology or bioinformatics is desirable.
</ul>
        		
	<p>Strong ability in statistical analysis or computer programming (python/perl, C/C++, R, Matlab) is desirable.</p>
     The postdoctoral research scholars will have the opportunity to collaborate with internationally recognized biomedical scientists at University of Iowa Carver College of Medicine, Stanford University, University of Oxford and Pacific Biosciences.

	<br>    
    </ul>

    <p>
    	<br>
    </p>


 <h2>Latest Publications</h2>


<p>
  <br><b>Single cell expression analysis of primate-specific retroviruses-derived HPAT lincRNAs in viable human blastocysts identifies embryonic cells co-expressing genetic markers of multiple lineages.</b><br>
  Glinsky, G., Durruthy-Durruthy, J., Wossidlo, M., Grow, E.J., <b>Weirather, J.L.</b>,<b> Au, K.F.</b>, Wysocka, J., Sebastiano, V.
  <br><i> Heliyon.</i> 2018.  [<a href="https://www.sciencedirect.com/science/article/pii/S2405844017328153">Manuscript</a>]
<br>
  </p>



<p>
  <br><b>Revealing tumor heterogeneity of breast cancer by utilizing the linkage between somatic and germline mutations.</b><br>
  <b>Zou, M., Jin, R., Au, K.F.#</b>
  <br><i> Briefings in Bioinformatics.</i> 2018. (in press) [<a href="https://www.healthcare.uiowa.edu/labs/au/GenoClone/">Link</a>]
<br><b># Corresponding author</b>
  </p>



<p><span lang=EN style='mso-ansi-language:EN'><br>
<b>IDP-denovo: <i>de novo</i> transcriptome assembly and isoform annotation by hybrid sequencing.</b><br>
<b>Fu, S.</b>, <b>Ma Y.</b>, Yao, H., Xu, Z., Chen, S., Song, J., <b>Au, K.F.</b> <br>
<i>Bioinformatics.</i> 2018. [<a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty098/4904264">Manuscript</a>]
<o:p></o:p></span></p>



<p>
  <br><b>Hybrid Sequencing of Full-Length cDNA Transcripts of Stems and Leaves in Dendrobium officinale.</b><br>
  He, L., <b>Fu, S. </b>, Xu, Z., Yan, J., Xu, J., Zhou, H., Zhou, J., Chen, X., Li, Y., <b>Au, K.F.<sup>#</sup></b>, Yao, H.<sup>#</sup>
  <br><i>Genes.</i> 2017. [<a href="http://www.mdpi.com/2073-4425/8/10/257">Manuscript</a>]
<br><b># Corresponding authors</b>
  </p>


<p>
  <br><b>CF airway smooth muscle transcriptome reveals a role for PYK2.</b><br>
  Cook, D.P., Adam, R.J., Zarei, K., <b>Deonovic, B.</b>, Stroik, M.R., Gansemer, N.D., Meyerholz, D.K., <b>Au, K.F.</b>, Stoltz, D.A.
  <br><i>JCI Insight.</i> 2017. [<a href="https://insight.jci.org/articles/view/95332">Manuscript</a>]
  </p>




    <p>
    <br><b>Comprehensive comparison of Pacific Biosciences and Oxford Nanopore Technologies and their applications to transcriptome analysis [version 2; referees: 2 approved].</b>
    <br><b>Weirather, J.L.*</b>, de Cesare, M.*, <b>Wang, Y.*</b>, Piazza, P., Vittorio, S., Wang, X.J., Buck, D., <b>Au, K.F.</b>
    <br><i>F1000Research</i> 2017, 6:100 [<a href="https://f1000research.com/articles/6-100/v2">Manuscript</a>]
    <br><b>* These authors contributed equally to this work</b>
    </p>


<p>
  <br><b>Gaining comprehensive biological insight into the transcriptome by performing a broad-spectrum RNA-seq analysis.</b><br>
  Sahraeian, S.M.E., Mohiyuddin, M., Sebra, R., Tilgner, H., Afshar P.T., <b>Au, K.F. </b>, Bani Asadi, N., Gerstein, M.B., Wong, W.H., Snyder, M.P., Schadt, E., Lam, H.Y.K.
  <br><i>Nature Communications.</i> 2017. [<a href="https://www.nature.com/articles/s41467-017-00050-4">Manuscript</a>]
  </p>


<p>
    	<br>
    </p>



   
	<h2>Research Focus</h2> </p>
    <ul>
	<li><b>RNA-seq and Third Generation Sequencing</b></li>
		<ul> We are interested in methodology research of Third Generation Sequencing (TGS) (especially for PacBio and Oxford Nanopore sequencing). 
		Au lab is working on both hybrid sequencing (Second Generation Sequencing (SGS) + TGS) and TGS-alone methodology research.
		Our research interests include but not limited to alternative splicing, isoform construction, gene fusion and quantitative analysis.</ul>
		<li><b>Stem cell transcriptome analysis</b></li>
		<ul>Au lab is applying the hybrid sequencing method on ESC, iPSC and preimplantation embryo, to deeply study the transcriptome differences between stem cells. </ul>
        <li><b>Proteomics</b> </li>
		<ul>Protein identification and novel splice detections from tandem Mass Spec are our research interests. Au lab is developing statistical methods for Integration of Mass Spec and sequencing data, in order to solve difficult proteomics problems.</ul>
		
	In addition, we are interested in the methodology research of gene regulatory network and RNA editing. 
	<br>    
    </ul>

    <p>
    	<br>
    </p>

	<h2>Collaborators</h2>
	Currently, we have very close ongoing collobrations with: 
	<ul><a href="http://www.stanford.edu/group/wonglab/">Wing H. Wong's Lab</a> at Stanford<br>
	<a href="http://www.pacificbiosciences.com/">Pacific Biosciences (PacBio)</a><br>
	<a href="http://www.binatechnologies.com/">Bina Technologies</a><br>
	<a href="http://stemcellphd.stanford.edu/faculty/renee-reijo-pera.html">Renee Reijo Pera's Lab</a> at Stanford<br>
    <a href="http://www.houstonmethodist.org/wing-tak-jack-wong-phd">Jack H. Wong's Lab</a> at Houston Methodist Research Institute<br>
    <a href="http://stemcellphd.stanford.edu/faculty/vittorio-sebastiano.html">Vittorio Sebastiano's Lab</a> at Stanford<br>
    <a href="http://www.well.ox.ac.uk/ogc/about-ogc">David Buck and the Oxford Genomics Centre</a> at the Wellcome Trust Centre for Human Genetics<br>
<a href="http://omicslab.genetics.ac.cn/index_en.php/">Xiujie Wang's Lab</a> at Beijing Institute of Genomics, Chinese Academy of Sciences<br>


<a href="http://sourcedb.big.cas.cn/yw/pl/fs/BIG_GPM/200907/t20090724_2194398.html">Yungui Yang's Lab</a> at Institute for Genetics and Developmental Biology, Chinese Academy of Sciences<br>

<a href="https://zhaohua.urmc.rochester.edu/">Xin Zhiguo Li's Lab</a> at University of Rochester<br>



    </ul>

    <p>
    	<br>
    </p>

   




   

	
    <p>
    	<br>
    </p>

    <h2>News</h2>
    <div>


<h3><img src="images/dna.png"> 06-15-2018: Shuhua Fu, Anqi Wang and Andrew Thurman are going to make presentations in ISMB 2018, July 7th, Chicago. </h3>
    <b>Topics:</b>
    <br>IDP-denovo: <i>de novo</i> transcriptome assembly and isoform annotation by hybrid sequencing. <b>(Shuhua Fu)</b>
    <br>Theoretical analysis of graph-based and alignment-based hybrid error correction methods for error-prone long reads. <b>(Anqi Wang)</b>
    <br>Gene isoform abundance quantification with Third Generation transcriptome sequencing. <b>(Andrew Thurman)</b>


<h3><img src="images/dna.png"> 02-23-2018: IDP-denovo paper is published </h3>
    <b>IDP-denovo: <i>de novo</i> transcriptome assembly and isoform annotation by hybrid sequencing.</b>
    <br><b>Fu, S.</b>, <b>Ma Y.</b>, Yao, H., Xu, Z., Chen, S., Song, J., <b>Au, K.F.</b> 
    <br><i>Bioinformatics</i> 2018. [<a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty098/4904264">Manuscript</a>]


    <h3><img src="images/dna.png"> 02-07-2016: AlignQC paper is released </h3>
    <b>Comprehensive comparison of Pacific Biosciences and Oxford Nanopore Technologies and their applications to transcriptome analysis [version 1; referees: awaiting peer review].</b>
    <br><b>Weirather, J.L.*</b>, de Cesare, M.*, <b>Wang, Y.*</b>, Piazza, P., Vittorio, S., Wang, X.J., Buck, D., <b>Au, K.F.</b>
    <br><i>F1000Research</i> 2017, 6:100 [<a href="http://dx.doi.org/10.12688/f1000research.10571.1">Manuscript</a>]
    <br><b>* These authors contributed equally to this work</b>
    <br>You can also check out the <a href="AlignQC/">Latest AlignQC Software</a> here if you are interested in using it to assess long read alignments.

    </div>
    <div>

    <h3><img src="images/dna.png"> 11-28-2016: IDP-ASE paper is released </h3>
    <b>IDP-ASE: haplotyping and quantifying allele-specific expression at the gene and gene isoform level by hybrid sequencing.</b><br>
    Deonovic, B., Wang, Y., Weirather, J., Wang, X.J., Au, K.F.
    <br><i>Nucleic Acids Research.</i> 2016. [<a href="http://nar.oxfordjournals.org/content/early/2016/11/28/nar.gkw1076">Manuscript</a>]

    <h3><img src="images/dna.png"> 07-08-2016: <a href="AlignQC/">AlignQC</a>, a tool for analyzing alignment files and assessing errors in alignments, is released.</h3>
    </div>
    
   
   
    
   
	<ul>
	
    </ul>
</p>
 
  </div>
  
<div id="sidebar">
    <h2>Site Map</h2>
    <ul>
      <li><a href="http://www.augroup.org">Home Page</a></li>
      <li><a href="http://www.augroup.org/labs/au/publications">Publications</a></li>
      <li><a href="http://www.augroup.org/labs/au/labmembers">Lab Members</a></li>
      <li><a href="http://www.augroup.org/labs/au/utility_script.asp">Utilities</a></li>
      <li><a href="http://www.augroup.org/labs/au/contact/">Contact</a></li>
    </ul>

    <h2>Software</h2>
    <ul>

    <li><a href="http://www.healthcare.uiowa.edu/labs/au/IDP/">IDP</a> - Isoform Detection and Prediction
      <br><a href="http://www.healthcare.uiowa.edu/labs/au/IDP/IDP_download_current.asp"><img src="images/download.png">IDP 0.1.8</a> (<a href="http://www.healthcare.uiowa.edu/labs/au/IDP/IDP_download_current.asp#0.1">notes</a>)</li>

    <li><a href="http://www.healthcare.uiowa.edu/labs/au/IDP-ASE/">IDP-ASE</a> - IDP for Allele-Specific Expression
      <br><a href="http://www.healthcare.uiowa.edu/labs/au/IDP-ASE/IDP-ASE_download_current.asp"><img src="images/download.png">IDP-ASE 20161017</a> (<a href="http://www.healthcare.uiowa.edu/labs/au/IDP-ASE/IDP-ASE_download_current.asp#0.1">notes</a>)</li>
          
    <li><a href="http://www.healthcare.uiowa.edu/labs/au/IDP-fusion/">IDP-fusion</a> - Fusion Isoform Detection and Prediction
      <br><a href="http://www.healthcare.uiowa.edu/labs/au/IDP-fusion/"><img src="images/download.png">IDP-fusion 1.1.1 </a> (<a href="http://www.healthcare.uiowa.edu/labs/au/IDP-fusion/IDP-fusion_download_current.asp#0.1">notes</a>)</li>
      
 
    <li><a href="http://www.healthcare.uiowa.edu/labs/au/LSC/">LSC</a> - Rapid error correction tool for PacBio data 
      <br><a href="http://www.healthcare.uiowa.edu/labs/au/LSC/LSC_download.asp"><img src="images/download.png">LSC 2.0</a> (<a href="http://www.healthcare.uiowa.edu/labs/au/LSC/LSC_download.asp#2.0">notes</a>)</li>
    
      <li><a href="http://www.stanford.edu/group/wonglab/SpliceMap/">SpliceMap</a> - De novo splice junction discovery and alignment tool 
    <br><a href="http://www.stanford.edu/group/wonglab/SpliceMap/download.html"><img src="images/download.png"> SpliceMap 3.3.5.2</a> (<a href="http://www.stanford.edu/group/wonglab/SpliceMap/download.html#3.3.5.2">notes</a>)</li>

      <li><a href="http://www.healthcare.uiowa.edu/labs/au/AlignQC/AlignQC_download_current.asp">AlignQC</a> - Alignment quality control reporting tool 
    <br><a href="http://www.healthcare.uiowa.edu/labs/au/AlignQC/AlignQC_download_current.asp"><img src="images/download.png"> AlignQC 1.2</a> (<a href="http://www.healthcare.uiowa.edu/labs/au/AlignQC/AlignQC_download_current.asp#notes">notes</a>)</li>
    

<li><a href="http://www.healthcare.uiowa.edu/labs/au/IDP-denovo/IDP-denovo_download_current.asp">IDP-denovo</a> - <i>de novo</i> transcriptome assembly and annotation tool 
    <br><a href="http://www.healthcare.uiowa.edu/labs/au/IDP-denovo/IDP-denovo_download_current.asp"><img src="images/download.png"> IDP-denovo 0.2</a> (<a href="http://www.healthcare.uiowa.edu/labs/au/IDP-denovo/IDP-denovo_download_current.asp#notes">notes</a>)</li>


<li><a href="http://www.healthcare.uiowa.edu/labs/au/GenoClone/">GenoClone</a> - Reveal tumor heterogeneity
    <br><a href="http://www.healthcare.uiowa.edu/labs/au/GenoClone/GenoClone_download_current.asp"><img src="images/download.png"> GenoCone 0.1</a> (<a href="http://www.healthcare.uiowa.edu/labs/au/GenoClone/GenoClone_download_current.asp#notes">notes</a>)</li>





    </ul>
  
        <h2>Useful links</h2>
    <ul>
      <li><a href="http://www.augroup.org">Au Lab</a> - Au Lab homepage </li>
    </ul>
    <ul>
      <li><a href="http://www.stanford.edu/group/wonglab/SpliceMap/">SpliceMap</a> - Novel splice junction discovery</li>
    </ul>
    <ul>
      <li><a href="http://www.healthcare.uiowa.edu/labs/au/LSC/">LSC</a> - Error correction of PacBio long reads </li>
    </ul>
    <ul>
      <li><a href="http://www.healthcare.uiowa.edu/labs/au/IDP/">IDP</a> - Isoform Detection and Prediction  </li>
    </ul>
    <ul>
      <li><a href="http://www.healthcare.uiowa.edu/labs/au/IDP-ASE/">IDP-ASE</a> - Determine haplotypes and quantify allele specific expression  </li>
    </ul>
    <ul>
      <li><a href="http://www.healthcare.uiowa.edu/labs/au/IDP-fusion/">IDP-Fusion</a> - Isoform Detection and Prediction across fusion genes  </li>
    </ul>
    <ul>
      <li><a href="http://www.healthcare.uiowa.edu/labs/au/IDP/hESC.asp">hESC transcriptome</a> - hESC transcriptome by IDP </li>
    </ul>
    <ul>
      <li><a href="http://www.healthcare.uiowa.edu/labs/au/AlignQC/">AlignQC</a> - Check an alignment, assess error patterns, and make reports </li>
    </ul>
   

<ul>
      <li><a href="http://www.healthcare.uiowa.edu/labs/au/IDP-denovo/">IDP-denovo</a> - <i>de novo</i> transcriptome assembly and annotation</li>
    </ul>


<ul>
      <li><a href="http://www.healthcare.uiowa.edu/labs/au/GenoClone/">GenoClone</a> - Reveal tumor heterogeneity</li>
    </ul>


 <ul>
      <li><a href="https://github.com/jason-weirather/Au-public">Au lab scripts</a> - Public repository of Au lab scripts and utilities </li>
    </ul>


  <br><ul><center><img src="images/seal.jpg" width="75%" hight="75%"><center></ul>
</div><div id="footer">
    <ul>
      <li>&copy; <a href=" http://www.healthcare.uiowa.edu/labs/au/">Kin Fai Au</a>. All Rights Reserved.</li>
      <li><a href="http://www.uiowa.edu/">University of Iowa</a>, IA 52242.</li>
    </ul>
</div>
  
  

<!-- end #content -->
</body>


<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-23879262-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>


</html>
