
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <!-- include virtual="header.inc" -->

 <head>
  <title>IDP tutorial</title>
  
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<link rel="stylesheet" type="text/css" href="styles/flex_textfont_sans.css" media="all" />
<!-- SANDSTONE AND TILE TEMPLATE (FLEXIBLE-WIDTH, SANS-SERIF TEXT FONT VERSION) PROVIDED BY STANFORD UNIVERSITY INFORMATION TECHNOLOGY SERVICES FOR NON-DEPARTMENTAL OR UNOFFICIAL USE BY STANFORD FACULTY, STAFF AND STUDENTS. DESIGNED IN 2009 BY DAVE REAM, IT SERVICES. FOR POLICIES AND GUIDELINES ON ACCESSIBILITY, COPYRIGHT, AND ON THE USE OF STANFORD'S NAME AND TRADEMARKS, INCLUDING THE UNIVERSITY SEAL, SEE http://soap.stanford.edu, http://www.stanford.edu/group/identity, and http://adminguide.stanford.edu --> 
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-50750315-1', 'uiowa.edu');
  ga('send', 'pageview');

</script>
</head>


<body>
<!--BEGIN TOP RED BAR -->
<div id="top_navbar">
  <!-- OPTIONAL TOP NAVIGATION OR TEXT AREA INSIDE RED BAR; BAR EXPANDS HEIGHT AUTOMATICALLY TO ACCOMMODATE CONTENT -->
   <ul>
    <li>IDP-ASE IS DEVELOPED BY <a href="http://www.augroup.org/">KIN FAI AU</a> with help from collaborators</li>
  </ul>
  <!-- END OF OPTIONAL TOP RED BAR CONTENT -->
</div>
<!-- END TOP RED BAR -->



<div id="content_container">
<!-- PAGE TITLE AREA AND BOX -->
  <div id="page_title_area">
    <!-- STANDARD PAGE TITLE; NO DECORATIVE GRAPHIC -->
    <!-- <div id="page_title_leftcontent"> -->
    <div id="page_title_leftcontent">
      <h1>IDP-ASE</h1>
      <ul>
        <li>IDP-ASE is a tool for <b>haplotyping and quantifying allele-specific expression</b> <br> from Second Generation Sequencing and PacBio sequencing. 
      </ul>
    </div>
    
    <div id="page_title_rightcontent">
	<ul>


	    <li>&nbsp;</li>

		<li><img src="images/IDP-fusion.png" width="300" >&nbsp;&nbsp;&nbsp;&nbsp;</li>
		
	</ul>
    </div>

<p class="page_title_area_lower">Latest News:  IDP-ASE_20161017 is released <a href="default.asp#news">read more</a></p>
    
    
  </div>

<!-- END PAGE TITLE AREA AND BOX -->
<div id="content">
  <h2>Tutorial</h2>
  <div id="content">
  This setup guide is in progress. For now see the <a href="https://github.com/bdeonovic/IDPASE.jl">Github</a>.

  <h3><img src="images/dna.png">  <b>Step 1</b> - Prepare Gene level data</h3>
  <pre> mkdir ~/temp/; mkdir ~/gene_files; mkdir ~/isoform_files; mkdir ~/gene_out; mkdir ~/isoform_out;
  julia ~/.julia/v0.4/IDPASE/src/prep_runs.jl -a test_data/SGS.psl test_data/TGS.psl \
    -g test_data/TDRKH.gpd -v test_data/sim.vcf -q test_data/SGS.fq test_data/TGS.fq \ 
    -d ~/temp/ -c 1 -f 1 1 -o ~/gene_files/ -p sim
  </pre>
  where flag -a is space separated list of PSL files, -g is GPD file, -v is VCF file, -q is space separated list of FASTQ files, -d is a directory for intermediate output, -c is a space separated list of chromosomes of interest, -f is a space separated list of FASTQ formats corresponding to the FASTQ files of -q, where 1 indicates PHRED+33, and 2 indicates PHRED+64, and -o indicates output directory and output prefix (so in example /out/ is directory and output files will be prefixed by sim).


  <h3><img src="images/dna.png">  <b>Step 2</b> - Obtain individual run commands</h3>
  <pre> julia  ~/.julia/v0.4/IDPASE/scripts/phase_by_loci.jl -a ~/gene_files/ -o ~/gene_out/ -n SGS TGS \ 
          -m 1 0 0 1 1 1 -d ~/.julia/v0.4/IDPASE/scripts/ -p sim > to_run_curr.sh
  </pre>
  where -a is the -o flag from command in step 5, -o is an output directory, -n are unique names corresponding to PSL files, -m is a vector indicating which combinations of the seq data to use with IDPASE. In the above example 3 runs of IDPASE will be issued where 1 0 indicates SGS only, 0 1 indicates TGS only, and 1 1 indicates hybrid-Seq. p is the prefix specified in step 5. The output is a list of commands to run for each gene individually. The flag -d is the directory where the IDPASE scripts are stored.


  <h3><img src="images/dna.png">  <b>Step 3</b> - Run each gene level command</h3>
  <pre> bash to_run_curr.sh </pre>


  <h3><img src="images/dna.png">  <b>Step 4</b> - Concatenate gene level results</h3>
  <pre> find ~/gene_out/ -name "REAL*" | xargs cat > ~/gene_out/gene_results.txt </pre>

  <h3><img src="images/dna.png">  <b>Step 5</b> - Prepare isoform level data</h3>
  <pre> julia ~/.julia/v0.4/IDPASE/src/prep_runs.jl -a test_data/SGS.psl test_data/TGS.psl \
          -g test_data/TDRKH.gpd -v test_data/sim.vcf -q test_data/SGS.fq test_data/TGS.fq \
          -d ~/temp/ -c 1 -f 1 1 -o ~/isoform_files/ -p sim -l 100 -i -s -e -r ~/gene_out/gene_results.txt
  </pre>
  where -l is read length for short reads, -s to skip file pre-processing (if using same GPD/VCF files are gene level), -e to use estimated haplotypes from gene leve, otherwise will use information from VCF, asumming it is phased, -r is the gene level results file.

  <h3><img src="images/dna.png">  <b>Step 6</b> - Obtain individual run commands</h3>
  <pre> julia ~/.julia/v0.4/IDPASE/scripts/phase_isoforms_by_loci.jl -i ~/isoform_files/ \
          -o ~/isoform_out/ -b ~/.julia/v0.4/IDPASE/scripts -a -p sim > to_run_isofs.sh
  </pre>

  <h3><img src="images/dna.png">  <b>Step 7</b> - Run each isoform level command</h3>
  <pre> bash to_run_isofs.sh </pre>

  <h3><img src="images/dna.png">  <b>Step 8</b> - Concatenate isoform level results</h3>
  <pre> find ~/isoform_out/ -name "EXTRA*" | xargs cat > ~/isoform_results.txt</pre>
</div>

</div>
  

 
<div id="sidebar">
    <h2>Site Map</h2>
    <ul>
      <li><a href="default.asp">Home</a></li>
      
      <li><a href="IDP-ASE_download_current.html">Downloads</a></li>
      <li><a href="IDP-ASE_contact.html">Contact</a></li>
      <li><a href="IDP-ASE_requirements.html">Requirements</a></li>
      
  
    </ul>
    <h3>Help</h3>
    <ul>
      
     

      <li><a href="IDP-ASE_guide.html">Manual</a></li>
      <li><a href="IDP-ASE_tutorial.html">Tutorial</a></li>
      
    </ul>
    <h2>Download</h2>
    <h3>Current release</h3>
    <ul>
      <li><a href="IDP-ASE_download_current.html"><img src="images/download.png"> IDP-ASE 20161017 </a> (<a href="IDP-ASE_download_current.asp#notes">notes</a>)</li>
    </ul>

        <h2>Useful links</h2>
    <ul>
      <li><a href="http://www.augroup.org">Au Lab</a> - Au Lab homepage </li>
    </ul>
    <ul>
      <li><a href="http://www.stanford.edu/group/wonglab/SpliceMap/">SpliceMap</a> - Novel splice junction discovery</li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/LSC/LSC">LSC</a> - Error correction of PacBio long reads </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP/IDP">IDP</a> - Isoform Detection and Prediction  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP-ASE/IDP-ASE">IDP-ASE</a> - Determine haplotypes and quantify allele specific expression  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP-fusion/IDP-fusion">IDP-Fusion</a> - Isoform Detection and Prediction across fusion genes  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP/hESC.html">hESC transcriptome</a> - hESC transcriptome by IDP </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/AlignQC/alignqc">AlignQC</a> - Check an alignment, assess error patterns, and make reports </li>
    </ul>
   

<ul>
      <li><a href="http://www.augroup.org/IDP-denovo.20160829/idp-denovo">IDP-denovo</a> - <i>de novo</i> transcriptome assembly and annotation</li>
    </ul>


<ul>
      <li><a href="http://www.augroup.org/GenoClone/GenoClone/GenoClone">GenoClone</a> - Reveal tumor heterogeneity</li>
    </ul>


 <ul>
      <li><a href="https://github.com/jason-weirather/Au-public">Au lab scripts</a> - Public repository of Au lab scripts and utilities </li>
    </ul>


    <br><ul><center><img src="images/seal.jpg" width="75%" hight="75%"><center></ul>
  </div>



<div id="footer">
    <ul>
      <li>&copy; <a href=" http://www.augroup.org">Kin Fai Au</a>. All Rights Reserved.</li>
      <li><a href="https://www.osu.edu/">The Ohio State University</a>, OH 43210.</li>
    </ul>
</div>
  
  

<!-- end #content -->
</body>

  <!-- include virtual="analytics.inc" -->

</html>
