<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <!-- include virtual="header.inc" -->

 <head>
  <title>GenoClone tutorial</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<link rel="stylesheet" type="text/css" href="styles/flex_textfont_sans.css" media="all" />
<!-- SANDSTONE AND TILE TEMPLATE (FLEXIBLE-WIDTH, SANS-SERIF TEXT FONT VERSION) PROVIDED BY STANFORD UNIVERSITY INFORMATION TECHNOLOGY SERVICES FOR NON-DEPARTMENTAL OR UNOFFICIAL USE BY STANFORD FACULTY, STAFF AND STUDENTS. DESIGNED IN 2009 BY DAVE REAM, IT SERVICES. FOR POLICIES AND GUIDELINES ON ACCESSIBILITY, COPYRIGHT, AND ON THE USE OF STANFORD'S NAME AND TRADEMARKS, INCLUDING THE UNIVERSITY SEAL, SEE http://soap.stanford.edu, http://www.stanford.edu/group/identity, and http://adminguide.stanford.edu --> 
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-50750315-1', 'uiowa.edu');
  ga('send', 'pageview');
</script>
</head>


<body>
<!--BEGIN TOP RED BAR -->
<div id="top_navbar">
  <!-- OPTIONAL TOP NAVIGATION OR TEXT AREA INSIDE RED BAR; BAR EXPANDS HEIGHT AUTOMATICALLY TO ACCOMMODATE CONTENT -->
   <ul>
    <li>GenoClone IS DEVELOPED BY <a href="http://www.augroup.org/">KIN FAI AU</a> with help from collaborators</li>
  </ul>
  <!-- END OF OPTIONAL TOP RED BAR CONTENT -->
</div>
<!-- END TOP RED BAR -->

<div id="content_container">
  <!-- PAGE TITLE AREA AND BOX -->
    <div id="page_title_area">
    <!-- STANDARD PAGE TITLE; NO DECORATIVE GRAPHIC -->
    <!-- <div id="page_title_leftcontent"> -->
    <div id="page_title_leftcontent">
      <h1>GenoClone</h1>
      <ul>
        <li>GenoClone is a tool for <b>subclone inference <br></b>and studying <b>tumor heterogenity</b> 

      </ul>
    </div>
    
    <div id="page_title_rightcontent">
	<ul>


	    <li>&nbsp;</li>

		<li><img src="images/GenoClone.png" width="500" >&nbsp;&nbsp;&nbsp;&nbsp;</li>
		
	</ul>
    </div>

<p class="page_title_area_lower">Latest News:  GenoClone is released <a href="default.asp#news">read more</a></p>
    
    
  </div>
  <!-- END PAGE TITLE AREA AND BOX -->
  <div id="content">
    <h2>Tutorial</h2>

    <p>This tutorial will help you get started with GenoClone by demonstrating how to perform <i>subclone inference</i> from exome or whole genome sequencing. If you experience any problems 
    following these steps, please don't hesitate to <a href="GenoClone_contact.html">contact</a> us. </p>

    <h3>Requirements</h3>
    <p>Please note the required software and data files to run GenoClone (<a href="GenoClone_requirements.html">requirements</a>). For the purpose of the tutorial we will make the required files available to you.<br>
    
    <h3><img src="images/dna.png">  <b>Download</b></h3>
    <p>You need to download:</p>
    <ul>
   <a href="GenoClone_download_current.asp"><img src="images/download.png"><b>GenoClone-0.1</b></a><br>
   <p>Uncompress and extract the directory in a location of your choice to get a directory containing GenoClone executable files.
   </p>
    <blockquote>
    <pre>
         $ tar -xzvf GenoClone-0.1.tar.gz
	 $ ls GenoClone/bin/
	 GenoClone
    </pre>
    </blockquote> 
	
	<p>Then check the commond of GenoClone.
   </p>
    <blockquote>
    <pre>
     $ GenoClone/bin/GenoClone -h
     <b>usage</b>: GenoClone [-h] -o OUTPUT
                 [--tempdir TEMPDIR | --specific_tempdir SPECIFIC_TEMPDIR]
                 varscan bam

     Detect the linkage information between somatic mutation(SNV) and germline
     mutation(SNP)

     <b>positional arguments</b>:
     varscan               REQUIRED Input the output file from Varscan
     bam                   REQUIRED Input the alignment tumor bam file

     <b>optional arguments</b>:
     -h, --help            show this help message and exit
     -o OUTPUT, --output OUTPUT
                        REQUIRED Output filename, totally it produce two
                        files, one '.csv' file for the composition of subclone
                        and the other '.pdf' file for the evaluation of
                        different number of subclones (default: None)
     --tempdir TEMPDIR  The temporary directory is made and destroyed here.
                        (default: /tmp)
     --specific_tempdir SPECIFIC_TEMPDIR
                        This temporary directory will be used, but will remain
                        after executing. (default: None)
    </pre>
    </blockquote
	
    <h3><img src="images/dna.png">  <b>Input data</b></h3>
    <p> GenoClone depends on the output of VarScan, the two softwares require: </p>
    <li>1. <b>Reference.fasta</b>: the indexed reference genome FASTA file.</li>
	<li>2. <b>Tumor.bam </b> and <b> Normal.bam </b>: the binary sequence alignment/map formatted (BAM) sequence data from tumor and matched normal DNA sample.</li>
	<p>
	</p>
	<p> VarScan needs the BAM file of tumor and matched normal and reference genome to obtain the total mutations (germline and somatic mutations). Then GenoClone uses the tumor bam file and total mutations
	to infer the subclones.</p>
	<p> The bam files for tumor and matched normal could be downloaded from <a href="https://gdc-portal.nci.nih.gov/search/s?facetTab=cases" >National Cancer Institute</a>
	and corresponding <a href="https://gdc.cancer.gov/about-data/data-harmonization-and-generation/gdc-reference-files" >Reference.fasta</a> </p>
    
    <h3><img src="images/dna.png">  <b>Example commonds</b></h3>
    <p><b> Step 1.</b> Use VarScan to obtain the total mutations:
	</p>
	<blockquote>
    <pre>
          $ samtools mpileup -q 1 -f Reference.fasta Tumor.bam >tumor.pileup
	  $ samtools mpileup -q 1 -f Reference.Genome Normal.bam >tumor.pileup
	  $ java -jar varScan somatic normal.pileup tumor.pileup --output-vcf 1 --output-snp total_mutations.vcf
    </pre>
    </blockquote> 
	<p><b> Step 2.</b> Use GenoClone to infer subclones:
	</p>
	<blockquote>
    <pre>
          $ /GenoClone/bin/GenoClone total_mutations.vcf Tumor.bam -o Tumor
    </pre>
    </blockquote> 
	
	
    <h3><img src="images/dna.png"> <b>Output files</b></h3>
   <p>GenoClone output two files:</p>
   <li> <b>Tumor.csv</b>: the number of subclones and the compositon of each subclones and the goodness value varies with number of subclones.
   </li>
   <li> <b>Tumor.pdf</b>: the difference between true and observed VAF and goodness varies with number of subclones.
   </li>
    
  </div>
<div id="sidebar">
    <h2>Site Map</h2>
    <ul>
      <li><a href="default.asp">Home</a></li>
      
      <li><a href="GenoClone_download_current.html">Downloads</a></li>
      <li><a href="GenoClone_contact.html">Contact</a></li>
      <li><a href="GenoClone_requirements.html">Requirements</a></li>
      
  
    </ul>
    <h3>Help</h3>
    <ul>
      <li><a href="GenoClone_manual.html">Manual</a></li>
      <li><a href="GenoClone_tutorial.html">Tutorial</a></li>
      
    </ul>
    <h2>Download</h2>
    <h3>Current release</h3>
    <ul>
      <li><a href="GenoClone_download_current.html"><img src="images/download.png"> GenoClone 0.1</a> (<a href="GenoClone_download_current.html#notes">notes</a>)</li>
    </ul>

<h2>Useful links</h2>
    <ul>
      <li><a href="http://www.augroup.org">Au Lab</a> - Au Lab homepage </li>
    </ul>
    <ul>
      <li><a href="http://www.stanford.edu/group/wonglab/SpliceMap/">SpliceMap</a> - Novel splice junction discovery</li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/LSC/LSC">LSC</a> - Error correction of PacBio long reads </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP/IDP">IDP</a> - Isoform Detection and Prediction  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP-ASE/IDP-ASE">IDP-ASE</a> - Determine haplotypes and quantify allele specific expression  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP-fusion/IDP-fusion">IDP-Fusion</a> - Isoform Detection and Prediction across fusion genes  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP/hESC.html">hESC transcriptome</a> - hESC transcriptome by IDP </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/AlignQC/alignqc">AlignQC</a> - Check an alignment, assess error patterns, and make reports </li>
    </ul>
	
<ul>
      <li><a href="http://www.augroup.org/IDP-denovo.20160829/idp-denovo">IDP-denovo</a> - <i>de novo</i> transcriptome assembly and annotation</li>
    </ul>


<ul>
      <li><a href="http://www.augroup.org/GenoClone/GenoClone/GenoClone">GenoClone</a> - Reveal tumor heterogeneity</li>
    </ul>

 <ul>
      <li><a href="https://github.com/jason-weirather/Au-public">Au lab scripts</a> - Public repository of Au lab scripts and utilities </li>
    </ul>

    <br><ul><center><img src="images/seal.jpg" width="75%" hight="75%"><center></ul>
  </div>
<div id="footer">
    <ul>
      <li>&copy; <a href=" http://www.augroup.org/">Kin Fai Au</a>.  All Rights Reserved.</li>
      <li><a href="https://www.osu.edu/">The Ohio State University</a>, OH 43210.</li>
    </ul>
</div>


<!-- end #content -->
</body>

  <!-- include virtual="analytics.inc" -->

</html>
