<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<head>
  <title>Long read files</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<link rel="stylesheet" type="text/css" href="styles/flex_textfont_sans.css" media="all" />
<!-- SANDSTONE AND TILE TEMPLATE (FLEXIBLE-WIDTH, SANS-SERIF TEXT FONT VERSION) PROVIDED BY STANFORD UNIVERSITY INFORMATION TECHNOLOGY SERVICES FOR NON-DEPARTMENTAL OR UNOFFICIAL USE BY STANFORD FACULTY, STAFF AND STUDENTS. DESIGNED IN 2009 BY DAVE REAM, IT SERVICES. FOR POLICIES AND GUIDELINES ON ACCESSIBILITY, COPYRIGHT, AND ON THE USE OF STANFORD'S NAME AND TRADEMARKS, INCLUDING THE UNIVERSITY SEAL, SEE http://soap.stanford.edu, http://www.stanford.edu/group/identity, and http://adminguide.stanford.edu --> 
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-50750315-1', 'uiowa.edu');
  ga('send', 'pageview');
</script>
</head>
<body>

<!--BEGIN TOP RED BAR -->
<div id="top_navbar">
  <!-- OPTIONAL TOP NAVIGATION OR TEXT AREA INSIDE RED BAR; BAR EXPANDS HEIGHT AUTOMATICALLY TO ACCOMMODATE CONTENT -->
   <ul>
    <li>IDP-denovo IS DEVELOPED BY <a href="http://www.augroup.org/">KIN FAI AU</a> with help from collaborators</li>
  </ul>
  <!-- END OF OPTIONAL TOP RED BAR CONTENT -->
</div>
<!-- END TOP RED BAR -->
 
<div id="content_container">
  <!-- PAGE TITLE AREA AND BOX -->
 <div id="page_title_area">
    <!-- STANDARD PAGE TITLE; NO DECORATIVE GRAPHIC -->
    <!-- <div id="page_title_leftcontent"> -->
    <div id="page_title_leftcontent">
      <h1>IDP-denovo</h1>
      <ul>
        <li>IDP-denovo is a tool for <b><i>de novo</i></b> transcriptome assembly and isoform annotation by hybrid sequencing.</li> 
      </ul>
    </div>
    
    <div id="page_title_rightcontent">
	<ul>


	    <li>&nbsp;</li>

		<li><img src="images/IDP-denovo.png" width="220" height="200">&nbsp;&nbsp;&nbsp;&nbsp;</li>
		
	</ul>
    </div>

<p class="page_title_area_lower">Latest News:  IDP-denovo_v.1 is released <a href="idp-denovo.html#news">read more</a></p>
    
    
  </div>

  <!-- END PAGE TITLE AREA AND BOX -->
  <div id="content">
  Download and install IDP-denovo, read "readme" in code package for details. 
  <pre>
   make
  </pre>

<b> For impatient people </b>
<pre> 
   ./idpdenovo  <options>
   </pre>
e.g, 
./idpdenovo scaffold=/user/shuhua/idpdenovo/test_data/scaffold.fa lr=/user/shuhua/idpdenovo/test_data/lr.fa leftSr=/user/shuhua/idpdenovo/test_data/leftSr.fa rightSr=/user/shuhua/idpdenovo/test_data/rightSr.fa

For test (in directory "test_data"): 
"scaffold.fa", "lr.fa" "leftSr.fa" "rightSr.fa"

  <h3><img src="images/dna.png">  <b>Step 1</b> - SR-scaffold extension and k-mer cluster</h3>
  <pre>
        scaffoldExtend <LR align to scaffold.sam> <scaffold.fa> <LR.fa>
         </pre>
    scaffold.fa is from Oases output (transcripts.fa), In "LR.fa", tags of sequences format are "lr1,lr2..."
  <pre>
       kCluster <seq.fa> <k-length> <k-mer percentage cutoff>
        </pre>
   To cluster unaligned LRs with k-mer clustering method
   Output includes a TXT file and a Log file

  <h3><img src="images/dna.png">  <b>Step 2</b> - Isoform annotation</h3>
  <pre> kCluster <unaligned LR.fa> kmer_size > clusterOutput 2> clusterLog
         </pre>

  To cluster unaligned LRs with k-mer clustering method, output a TXT file and a log file to track progress of clustering

  <h3><img src="images/dna.png">  <b>Step 3</b> - Quantification </h3>
  <pre> MLE <SR align to pseudo-ref SAM output> <GPD file> python_path code_path SR_length  </pre>
   Output file is "refSeq_MLE_output1.tab" to tell SR coverage for each transcript.
  IDP-denovo will outupt LR coverage.

  SR align to pseudo-reference with HISAT 
  LR align to transcripts with GMAP
  
<div id="sidebar">
    <h2>Site Map</h2>
    <ul>
      <li><a href="idp-denovo.html">Home</a></li>
      
      <li><a href="IDP-denovo_download_current.html">Downloads</a></li>
      <li><a href="IDP-denovo_contact.html">Contact</a></li>
      <li><a href="IDP-denovo_requirements.html">Requirements</a></li>
      <li><a href="IDP-denovo_howitworks.html">How it works?</a></li>

  <li><a href="IDP-denovo_faq.html">FAQ</a></li>

    </ul>
    <h3>Help</h3>
    <ul>
      
     

      <li><a href="IDP-denovo_guide.html">Manual</a></li>
      <li><a href="IDP-denovo_tutorial.html">Tutorial</a></li>
      
    </ul>
    <h2>Download</h2>
    <h3>Current release</h3>
    <ul>
      <li><a href="IDP-denovo_download_current.html"><img src="images/download.png"> IDP-denovo 0.2 </a> (<a href="IDP-denovo_download_current.html#notes">notes</a>)</li>
    </ul>

        <h2>Useful links</h2>
    <ul>
      <li><a href="http://www.augroup.org/">Au Lab</a> - Au Lab homepage </li>
    </ul>
    <ul>
      <li><a href="http://www.stanford.edu/group/wonglab/SpliceMap/">SpliceMap</a> - Novel splice junction discovery</li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/LSC/LSC">LSC</a> - Error correction of PacBio long reads </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP/IDP">IDP</a> - Isoform Detection and Prediction  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP-ASE/IDP-ASE">IDP-ASE</a> - Determine haplotypes and quantify allele specific expression  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP-fusion/IDP-fusion">IDP-Fusion</a> - Isoform Detection and Prediction across fusion genes  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP/hESC.html">hESC transcriptome</a> - hESC transcriptome by IDP </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/AlignQC/alignqc/">AlignQC</a> - Check an alignment, assess error patterns, and make reports </li>
    </ul>
   

<ul>
      <li><a href="http://www.augroup.org/IDP-denovo.20160829/idp-denovo/">IDP-denovo</a> - <i>de novo</i> transcriptome assembly and annotation</li>
    </ul>


<ul>
      <li><a href="http://www.augroup.org/GenoClone/GenoClone/">GenoClone</a> - Reveal tumor heterogeneity</li>
    </ul>


 <ul>
      <li><a href="https://github.com/jason-weirather/Au-public">Au lab scripts</a> - Public repository of Au lab scripts and utilities </li>
    </ul>


    <br><ul><center><img src="images/seal.jpg" width="75%" hight="75%"><center></ul>
  </div>



<div id="footer">
    <ul>
      <li>&copy; <a href="http://www.augroup.org/">Kin Fai Au</a>.  All Rights Reserved.</li>
      <li><a href="http://www.osu.edu">The Ohio State University</a>, OH 43210.</li>
    </ul>
</div>
  
<!-- end #content -->
</body>

  <!-- include virtual="analytics.inc" -->

</html>
