<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>hESC-specific novel genes</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<link rel="stylesheet" type="text/css" href="styles/flex_textfont_sans.css" media="all" />
<!-- SANDSTONE AND TILE TEMPLATE (FLEXIBLE-WIDTH, SANS-SERIF TEXT FONT VERSION) PROVIDED BY STANFORD UNIVERSITY INFORMATION TECHNOLOGY SERVICES FOR NON-DEPARTMENTAL OR UNOFFICIAL USE BY STANFORD FACULTY, STAFF AND STUDENTS. DESIGNED IN 2009 BY DAVE REAM, IT SERVICES. FOR POLICIES AND GUIDELINES ON ACCESSIBILITY, COPYRIGHT, AND ON THE USE OF STANFORD'S NAME AND TRADEMARKS, INCLUDING THE UNIVERSITY SEAL, SEE http://soap.stanford.edu, http://www.stanford.edu/group/identity, and http://adminguide.stanford.edu --> 
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-50750315-1', 'uiowa.edu');
  ga('send', 'pageview');
</script>
</head>

<body>
<!--BEGIN TOP RED BAR -->
<div id="top_navbar">
  <!-- OPTIONAL TOP NAVIGATION OR TEXT AREA INSIDE RED BAR; BAR EXPANDS HEIGHT AUTOMATICALLY TO ACCOMMODATE CONTENT -->
   <ul>
    <li>This hESC transcriptome iS identified by <a href="http://www.stanford.edu/group/wonglab/SpliceMap/">SpliceMap</a>-<a href="http://www.augroup.org/LSC/">LSC</a>-<a href="http://www.augroup.org/IDP/">IDP</a> pipeline</li>
  </ul>
  <!-- END OF OPTIONAL TOP RED BAR CONTENT -->
</div>
<!-- END TOP RED BAR -->
<div id="content_container">
  <!-- PAGE TITLE AREA AND BOX -->
<div id="page_title_area">
    <!-- STANDARD PAGE TITLE; NO DECORATIVE GRAPHIC -->
    <!-- <div id="page_title_leftcontent"> -->
    <div id="page_title_leftcontent">
      <h1>hESC transcriptome</h1>
      <ul>
        <li>This Human Embryonic Stem Cell <br>(hESC, H1 cell line) <br>transcriptome is identified by <a href="http://www.stanford.edu/group/wonglab/SpliceMap/"><b>SpliceMap</b></a>-<a href="http://www.augroup.org/LSC/"><b>LSC</b></a>-<a href="http://www.augroup.org/IDP/"><b>IDP</b></a> pipeline<br>

      </ul>
    </div>
    
    <div id="page_title_rightcontent">
  <ul>


      <li>&nbsp;</li>

    <li><img src="images/IDP.png" width="300" >&nbsp;&nbsp;&nbsp;&nbsp;</li>
    
  </ul>
    </div>
    
    
  </div>
  <!-- PAGE TITLE LEFT CONTENT WITH OPTIONAL QUAD LAMP DECORATION; UN-COMMENT TO USE -->


  <!-- END PAGE TITLE AREA AND BOX -->
  <div id="content">
    <h2> A). Definition of novel genes</h2>
    <p> According to the junction usage, we can categorized 2,428(325 + 2,103) novel multi-exon isoforms into four groups. In order to find completely novel multi-exon genes, we check the novelity of their junctions. <b>If all junctions of an predicted isoform are novel (i.e. not reported by RefSeq, Ensembl, KnonwGen or Gencode), it is defined as an isoform from a novel genes (the red group).</b>

	
	<center><img src="images/figure2AB.png" width="100%" hight="100%"> </center>
    <h3>Four groups of novel isoforms</h3>
    	 You can download these novel isoform groups at this webpage. All isoforms are divided as the Figure A below. The download links of different groups are highlighted in the same colors of the piechart</p>
	<br>

    <ul>
        <li><img src="images/download.png"><b><a href="data/hESC_Isoforms_of_Novel_Genes_v1.zip"><font color="brown"> Isoforms of novel genes (273)</font></a></b></li>
			<dd> ALL junctions in each isoform are NOT annotated by RefSeq, Ensembl, KnownGen or Gencode. </dd>	
		<li><img src="images/download.png"><b><a href="data/hESC_Novel_Isoforms_of_Exisiting_Genes_with_Novel_Junctions_v1.zip"><font color="orange"> Novel isoforms of exisiting genes with novel junctions (655)</font></a></b></li>
			<dd> AT LEAST ONE junctions in each isoform are NOT annotated by RefSeq, Ensembl, KnownGen or Gencode. </dd>	
		<li><img src="images/download.png"><b><a href="data/hESC_Novel_Combinations_of_Existing_Junctions_v1.zip"><font color="4CAEE3"> Isoforms that are novel combinations of existing junctions (876)</font></a></b></li>
			<dd> ALL junctions in each isoform are annotated by RefSeq, Ensembl, KnownGen or Gencode. But their combinations are NOT annotated by these libraries. </dd>	
		<li><img src="images/download.png"><b><a href="data/hESC_Fragment_of_Existing_Isoforms_v1.zip"><font color="#728FCE"> Fragment of existing isoforms (624)</font></a></b></li>
			<dd> The junction linkage of each isoform is part of the annotated junction linkages in RefSeq, Ensembl, KnownGen or Gencode. </dd>	
	</ul>
	    
    <h2>B). Relatively high expression in hESC (H1)</h2>
    273 <b><a href="data/hESC_Isoforms_of_Novel_Genes.zip"><font color="brown"> novel isoforms </font></a></b> are from 216 novel gene loci. The abundance of these 213 gene loci are compared between H1 and 16 human tissues by abundance ratio:
<br>
        <dd> <style>
    td.upper_line { border-top:solid 1px black; }
    table.fraction { text-align: center; vertical-align: middle;
        margin-top:0.5em; margin-bottom:0.5em; line-height: 2em; }
</style>

<table class="fraction" align="center" cellpadding="0" cellspacing="0">
    <tr>
        <td rowspan="2" nowrap="nowrap">
            <i>abundance ratio</i> <b>=  </b>
        </td><td nowrap="nowrap">
            gene abundance in human tissue
        </td>
    </tr><tr>
        <td class="upper_line">
            gene abundance in H1
        </td>
    </tr>
</table>
</dd>
	<ul>
	<li> RNA-seq data for 16 different human tissues (adipose, adrenal, brain, breast, colon, heart, kidney, liver, lung, lymph node, ovary, prostate, skeletal muscle, testes, thyroid, and white blood cells) were generated by the Illumina Human Body Map 2.0 Project</li>
	</ul>
	<p> The standard deviation and the mean of 16 abundance ratios of each gene locus were computed and ploted (Figure B). 146 novel genes (68%, inside the red box) have averaged abundance (among 16 human tissues) ratio smaller than 0.5 with standard deviation smaller than 0.5.
	Please find the RPKMs and the abundance ratios of these novel genes in H1 and 16 human tissues <b><a href="data/NovelGeneAbundanceRatio.xlsx">here</a></b>.</p><br>

	<h2>Human Pluripotent Associated Transcript (HPAT) </h2>
    <p> To confirm that the novel genes have expression specific to hESC, we further characterized 23 of those most expressed. For this purpose we examined expression by qPCR in two independent hESC lines (H1 and H9); one induced pluripotent stem cell line (iPSC line RiPSC.HUF1) and a collection of cDNAs obtained from different adult human tissues. All 23 targets, denoted as <b>HPAT (Human Pluripotent Associated Transcript)</b> genes, were abundantly expressed in the pluripotent cell lines (H1, H9, RiPSC.HUF1) as opposed to adult tissues where they were either undetectable or expressed at very low levels. </p>

	
	<center><img src="images/HPAT.png" width="100%" hight="100%"> </center><br>
	Y axis: relative gene expression (the ratio over the endogenous)<br>
	X axis (samples): 1. H1 cells, 2. iPSC - RIPSC.HUF1,  3.iPSC - RIPSC.BJ, 4.BJ fibroblasts, 5. Genomic control (gDNA), 6.NTC (non retrotranscribed)
	<br>
	<b>Note</b>: 
	<ul>
	<li><font color="blue">  The HPATs in blue has no/low expression in iPSC - RIPSC.BJ but high in H1 and iPSC - RIPSC.HUF1.</font></li>
	<li><font color="red">  The HPATs in blue has no/low expression in iPSC - RIPSC.HUF1 but high in H1 and iPSC - RIPSC.BJ.</font></li>
	<li><font color="orange">  The HPATs in blue has no/low expression in iPSC - RIPSC.BJ or iPSC - RIPSC.HUF1 but high in H1.</font></li>
	</ul>
	These may infer to the incompleteness of reprogramming of two iPSCs.
	<br>
	<br>
	
    <table border="1" cellpadding="2">
    <tr>
    <td> <b>HPAT names</b> </td>
     <td> <b>Original names in IDP output </b> </td>
    </tr>

    <tr>
    <td> <b>HPAT1</b> </td>
     <td> chr2:210164662-210175596 </td>
    </tr>

    <tr>
    <td> <b>HPAT2</b> </td>
     <td>chr2:67559194-67561084</td>
    </tr>	

    <tr>
    <td> <b>HPAT3</b> </td>
     <td>chr2:66471616-66473418</td>
    </tr>	

    <tr>
    <td> <b>HPAT4</b> </td>
     <td>chr2:8018612-8031407</td>
    </tr>	

    <tr>
    <td> <b>HPAT5</b> </td>
     <td> chr6:167641868-167659274 </td>
    </tr>	

    <tr>
    <td> <b>HPAT6</b> </td>
     <td>chr13:43417023-4344669</td>
    </tr>	

    <tr>
    <td> <b>HPAT7</b> </td>
     <td>chr1:114557791-114564550</td>
    </tr>	

    <tr>
    <td> <b>HPAT8</b> </td>
     <td> chr2:188898787-188903856</td>
    </tr>	

    <tr>
    <td> <b>HPAT9</b> </td>
     <td>chr3:154996348-154998035</td>
    </tr>	

    <tr>
    <td> <b>HPAT10</b> </td>
     <td>chr3:176018527-176036215</td>
    </tr>	

    <tr>
    <td> <b>HPAT11</b> </td>
     <td>chr3:191633329-191644765</td>
    </tr>	

    <tr>
    <td> <b>HPAT12</b> </td>
     <td>chr4:88843339-88852526</td>
    </tr>	

    <tr>
    <td> <b>HPAT13</b> </td>
     <td>chr4:93179108-93184341</td>
    </tr>	

    <tr>
    <td> <b>HPAT14</b> </td>
     <td>chr6:77271075-77271256 and chr6:77271384-77452727</td>
    </tr>	

    <tr>
    <td> <b>HPAT15</b> </td>
     <td>chr13:51744598-51761472</td>
    </tr>	

    <tr>
    <td> <b>HPAT16</b> </td>
     <td>chr13:54909278-54927276</td>
    </tr>	

    <tr>
    <td> <b>HPAT17</b> </td>
     <td>chr14:48703426-48704633</td>
    </tr>	

    <tr>
    <td> <b>HPAT18</b> </td>
     <td>chr14:48731686-48734763</td>
    </tr>	

    <tr>
    <td> <b>HPAT19</b> </td>
     <td>chr16:9761057-9761138</td>
    </tr>	

    <tr>
    <td> <b>HPAT20</b> </td>
     <td>chr19:23444546-23456877</td>
    </tr>	

    <tr>
    <td> <b>HPAT21</b> </td>
     <td>chr20:12735069-12740892</td>
    </tr>	

    <tr>
    <td> <b>HPAT22</b> </td>
     <td>chrX:124519261-124519663</td>
    </tr>	

    <tr>
    <td> <b>HPAT23</b> </td>
     <td>chr16:65261924-65268788</td>
    </tr>	
  
    </table>

	<br><p>  We keep optimizing the results by running the updated LSC and IDP. More and more reliable results will be updated. If you want to subscribe our update news, please contact Kin Fai Au: 	<img src="images/Picture3.png" width="20%" hight="20%"></p>
  
  </div>
<div id="sidebar">
    <h2>Site Map</h2>
    <ul>
      <li><a href="hESC.html">Home</a></li>
      <li><a href="hESC_Gene_isoform_detection_and_prediction.html">Gene isoform detection and prediction</a></li>
      <li><a href="hESC_hESC-specific_novel_genes.html">hESC-specific novel genes</a></li>
      <li><a href="hESC_Novel_gene_isoforms.html">Novel gene isoforms</a></li>
    <li><a href="hESC_Quantification_of_isoform_abundance.html">Quantification of isoform abundance</a></li>
    <li><a href="hESC_Isoforms_of_pluripotent_stem_cell_markers.html">Isoforms of pluripotent stem cell markers</a></li>
    <li><a href="hESC_ncRNA_identification.html">ncRNA identification</a></li>
    <li><a href="hESC_Data.html">Data</a></li>
    
    </ul>
    <h3>Help</h3>
    <ul>
    <li><a href="hESC_gpd_format.html">GPD format</a></li>
      <li><a href="hESC_howitworks.html">How it works?</a></li>
      <li><a href="http://www.augroup.org/IDP/hESC_long_read_file_guide.html">Generate long read files</a></li>
      <li><a href="http://www.augroup.org/utility_script.html">Utilities</a></li>
      <li><a href="hESC_faq.html">FAQ</a></li>
      <li><a href="hESC_contact.html">Contact</a></li>
    </ul>

        <h2>Useful links</h2>
       <ul>
      <li><a href="http://www.augroup.org">Au Lab</a> - Au Lab homepage </li>
    </ul>
    <ul>
      <li><a href="http://www.stanford.edu/group/wonglab/SpliceMap/">SpliceMap</a> - Novel splice junction discovery</li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/LSC/LSC">LSC</a> - Error correction of PacBio long reads </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP/IDP">IDP</a> - Isoform Detection and Prediction  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP-ASE/IDP-ASE">IDP-ASE</a> - Determine haplotypes and quantify allele specific expression  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP-fusion/IDP-fusion">IDP-Fusion</a> - Isoform Detection and Prediction across fusion genes  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP/hESC.html">hESC transcriptome</a> - hESC transcriptome by IDP </li>
    </ul>
    <ul>
      <li><a href="https://github.com/jason-weirather/Au-public">Au lab scripts</a> - Public repository of Au lab scripts and utilities </li>
    </ul>


    <br><ul><center><img src="images/seal.jpg" width="75%" hight="75%"><center></ul>
<div id="footer">
    <ul>
      <li>&copy; <a href=" http://www.augroup.org">Kin Fai Au</a>. All Rights Reserved.</li>
      <li><a href="https://www.osu.edu/">The Ohio State University</a>, OH 43210.</li>
    </ul>
</div>
<!-- end #content -->
</body>



</html>
