<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<head>
  <title>Gene isoform detection and prediction</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<link rel="stylesheet" type="text/css" href="styles/flex_textfont_sans.css" media="all" />
<!-- SANDSTONE AND TILE TEMPLATE (FLEXIBLE-WIDTH, SANS-SERIF TEXT FONT VERSION) PROVIDED BY STANFORD UNIVERSITY INFORMATION TECHNOLOGY SERVICES FOR NON-DEPARTMENTAL OR UNOFFICIAL USE BY STANFORD FACULTY, STAFF AND STUDENTS. DESIGNED IN 2009 BY DAVE REAM, IT SERVICES. FOR POLICIES AND GUIDELINES ON ACCESSIBILITY, COPYRIGHT, AND ON THE USE OF STANFORD'S NAME AND TRADEMARKS, INCLUDING THE UNIVERSITY SEAL, SEE http://soap.stanford.edu, http://www.stanford.edu/group/identity, and http://adminguide.stanford.edu --> 
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-50750315-1', 'uiowa.edu');
  ga('send', 'pageview');
</script>
</head>


<body>
<!--BEGIN TOP RED BAR -->
<div id="top_navbar">
  <!-- OPTIONAL TOP NAVIGATION OR TEXT AREA INSIDE RED BAR; BAR EXPANDS HEIGHT AUTOMATICALLY TO ACCOMMODATE CONTENT -->
   <ul>
    <li>This hESC transcriptome iS identified by <a href="http://www.stanford.edu/group/wonglab/SpliceMap/">SpliceMap</a>-<a href="http://www.augroup.org/LSC/">LSC</a>-<a href="http://www.augroup.org/IDP/">IDP</a> pipeline</li>
  </ul>
  <!-- END OF OPTIONAL TOP RED BAR CONTENT -->
</div>
<!-- END TOP RED BAR -->
<div id="content_container">
  <!-- PAGE TITLE AREA AND BOX -->
  <!-- #include file ="asp_hESC\titlearea.inc" -->


  <!-- PAGE TITLE LEFT CONTENT WITH OPTIONAL QUAD LAMP DECORATION; UN-COMMENT TO USE -->


  <!-- END PAGE TITLE AREA AND BOX -->
  <div id="content">
    <h2> Detection + Prediction = Identification</h2>
    <p>You can download the different part of the gene isoform identification from this webpage. All isoforms are divided as the figures below. The download links of different parts are highlighted in the same colors of the piechart</p>
	
	
	<center><img src="images/figure1A.png" width="100%" hight="100%"> </center>
    <h2>Gene isoform detection</h2>
    If there is a perfect junction linkage match of an exisiting annotated gene isoform (RefSeq in this study).in the the long reads, we call this isoform detected and this isoform is a <b>DETECTION</b>.
<br>

    <ul>
        <li><img src="images/download.png"><b><a href="data/hESC_detection_v1.zip"><font color="4CAEE3"> Gene isoform detection (8,084)</font></a></b></li>
			<dd> These 8,084 (2,732 + 5,352) detections are the gene isoforms annotated by RefSeq.</dd>	
	</ul>
	    
    <h2>Gene isoform prediction</h2>
    These relatively long isoforms are not detected directly (see Gene isoform detection). But we find their fragments from long reads. Based on these long reads, we applied Max A Posterior on short read coverage and then predict these isoforms at the FPR<0.05. Details are in <b><a href="hESC_howitworks.html">How it works?</a></b>
<br>
   <ul>
        <li><img src="images/download.png"><b><a href="data/hESC_Prediction_v1.zip"><font color="black"> Prediction (10,811)</font></a> = <font color="#728FCE"> Overlap of Prediction and Detection </font> + <font color="#FFD801"> Prediction only</font> </b></li>
			<dd> There are 10,811 (5,352 + 5,459) gene isoform predictions with FPR <0.1. 5,352 predictions are detected directly and 5,459 predictions are not detected by full-length. </dd>		
        <li><img src="images/download.png"><b><a href="data/hESC_RefSeqAnnotated_Prediction_NotDetection_v1.zip"><font color="#5D478B"> RefSeqAnnotated, Prediction only (1,785)</font></a></b></li>
			<dd> Out of 5,459 "<b><font color="#FFD801"> Prediction only</font></b>" isoforms, 1,785 are RefSeq annotated.</dd>	
		<li><img src="images/download.png"><b><a href="data/hESC_NonRefSeqAnnotated_Prediction_NotDetection_v1.zip"><font color="#3090C7"> NonRefSeqAnnotated, Prediction only (1,027)</font></a></b></li>
			<dd> Out of 5,459 "<b><font color="#FFD801"> Prediction only</font></b>" isoforms, 1,027 are not RefSeq annotated but annotated in Ensembl, KnownGen or Gencode. "NonRefSeq" is denoted as the combination of Ensembl, KnownGen and Gencode. </dd>		
        <li><img src="images/download.png"><b><a href="data/hESC_Novel_Isoform_v1.zip"><font color="#89C35C"> Novel isoform (2,103)</font></a></b></li>
			<dd> Out of 5,459 "<b><font color="#FFD801"> Prediction only</font></b>" isoforms, 2,103 are not reported by EST or annotated by RefSeq, Ensembl, KnownGen or Gencode. <a href="hESC_Novel_gene_isoforms.html">Please see more details in the section of "<b>Novel gene isoforms</b>"</a></dd>		

	</ul>
	    <p>  We keep optimizing the results by running the updated LSC and IDP. More and more reliable results will be updated. If you want to subscribe our update news, please contact Kin Fai Au: <img src="images/Picture3.png"> 

 
   
  </div>
<div id="sidebar">
    <h2>Site Map</h2>
    <ul>
      <li><a href="hESC.html">Home</a></li>
      <li><a href="hESC_Gene_isoform_detection_and_prediction.html">Gene isoform detection and prediction</a></li>
      <li><a href="hESC_hESC-specific_novel_genes.html">hESC-specific novel genes</a></li>
      <li><a href="hESC_Novel_gene_isoforms.html">Novel gene isoforms</a></li>
    <li><a href="hESC_Quantification_of_isoform_abundance.html">Quantification of isoform abundance</a></li>
    <li><a href="hESC_Isoforms_of_pluripotent_stem_cell_markers.html">Isoforms of pluripotent stem cell markers</a></li>
    <li><a href="hESC_ncRNA_identification.html">ncRNA identification</a></li>
    <li><a href="hESC_Data.html">Data</a></li>
    
    </ul>
    <h3>Help</h3>
    <ul>
    <li><a href="hESC_gpd_format.html">GPD format</a></li>
      <li><a href="hESC_howitworks.html">How it works?</a></li>
      <li><a href="http://www.augroup.org/IDP/hESC_long_read_file_guide.html">Generate long read files</a></li>
      <li><a href="http://www.augroup.org/utility_script.html">Utilities</a></li>
      <li><a href="hESC_faq.html">FAQ</a></li>
      <li><a href="hESC_contact.html">Contact</a></li>
    </ul>

        <h2>Useful links</h2>
       <ul>
      <li><a href="http://www.augroup.org">Au Lab</a> - Au Lab homepage </li>
    </ul>
    <ul>
      <li><a href="http://www.stanford.edu/group/wonglab/SpliceMap/">SpliceMap</a> - Novel splice junction discovery</li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/LSC/LSC">LSC</a> - Error correction of PacBio long reads </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP/IDP">IDP</a> - Isoform Detection and Prediction  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP-ASE/IDP-ASE">IDP-ASE</a> - Determine haplotypes and quantify allele specific expression  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP-fusion/IDP-fusion">IDP-Fusion</a> - Isoform Detection and Prediction across fusion genes  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP/hESC.html">hESC transcriptome</a> - hESC transcriptome by IDP </li>
    </ul>
    <ul>
      <li><a href="https://github.com/jason-weirather/Au-public">Au lab scripts</a> - Public repository of Au lab scripts and utilities </li>
    </ul>


    <br><ul><center><img src="images/seal.jpg" width="75%" hight="75%"><center></ul>
<div id="footer">
    <ul>
      <li>&copy; <a href=" http://www.augroup.org">Kin Fai Au</a>. All Rights Reserved.</li>
      <li><a href="https://www.osu.edu/">The Ohio State University</a>, OH 43210.</li>
    </ul>
</div><!-- end #content -->
</body>



</html>
