<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>LSC FAQ</title>

<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<link rel="stylesheet" type="text/css" href="styles/flex_textfont_sans.css" media="all" />
<!-- SANDSTONE AND TILE TEMPLATE (FLEXIBLE-WIDTH, SANS-SERIF TEXT FONT VERSION) PROVIDED BY STANFORD UNIVERSITY INFORMATION TECHNOLOGY SERVICES FOR NON-DEPARTMENTAL OR UNOFFICIAL USE BY STANFORD FACULTY, STAFF AND STUDENTS. DESIGNED IN 2009 BY DAVE REAM, IT SERVICES. FOR POLICIES AND GUIDELINES ON ACCESSIBILITY, COPYRIGHT, AND ON THE USE OF STANFORD'S NAME AND TRADEMARKS, INCLUDING THE UNIVERSITY SEAL, SEE http://soap.stanford.edu, http://www.stanford.edu/group/identity, and http://adminguide.stanford.edu --> 
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-50750315-1', 'uiowa.edu');
  ga('send', 'pageview');

</script>
</head>

<body>
 <!--BEGIN TOP RED BAR -->
<div id="top_navbar">
  <!-- OPTIONAL TOP NAVIGATION OR TEXT AREA INSIDE RED BAR; BAR EXPANDS HEIGHT AUTOMATICALLY TO ACCOMMODATE CONTENT -->
   <ul>
    <li><a href="http://www.augroup.org/">AU LAB - THE STATISTICAL BIOINFORMATICS LAB @ The Ohio State University</a></li>
  </ul>
  <!-- END OF OPTIONAL TOP RED BAR CONTENT -->
</div>
<!-- END TOP RED BAR -->


<div id="content_container">
  <!-- PAGE TITLE AREA AND BOX -->
<div id="page_title_area">
    <!-- STANDARD PAGE TITLE; NO DECORATIVE GRAPHIC -->
    <!-- <div id="page_title_leftcontent"> -->
    <div id="page_title_leftcontent">
      <h1>LSC</h1>
      <ul>
        <li>LSC is a long read error correction tool. <br>
        It offers fast correction with high sensitivity <br>
        and good accuracy. <br>

      </ul>
    </div>
    
    <div id="page_title_rightcontent">
	<ul>


	    <li>&nbsp;</li>

		<li><img src="images/LSC.png" width="300" >&nbsp;&nbsp;&nbsp;&nbsp;</li>
		
	</ul>
    </div>

<p class="page_title_area_lower">Latest News:  Major update version 2.0 ... <a href="LSC.html#news">read more</a></p>
    

  <!-- END PAGE TITLE AREA AND BOX -->
  <div id="content">
    <h2>FAQ</h2>
    If you have any questions that have not been answered here, please <a href="LSC_contact.html">contact</a> us.
    <h3><img src="images/dna.png"> What is LSC?</h3>
    LSC is a program written entirely in python2.6 that is able to correct the thrid generation long reads by the second generation 
    short reads. 
    <h3><img src="images/dna.png"> [ LSC 0.2.4 ] What is in the folder "output_old"?</h3>
	<ul>
	
	<li>Question:</li>
	I got three folders output, output_old, and temp.
I guess LSCoutput is new vision result and LSCoutput_old is earlier vision result. Right?
New result folder has new files like newLS.fa etc.
Could you explain the files mean? 
	</ul>
	<ul>

	<li>Answer:</li>
The folder output_old contains some "old" results which may have slightly lower accuracy.
In LSC 0.2.4, I corrected some bugs and output "new" results in the folder output. You should use these files. newLS.fa is just a temp file for debugging. you only need to use full_LR_SR.map.fa and corrected_LR_SR.map.fa
	</ul>


	
    <h3><img src="images/dna.png"> "awk" bugs?</h3>
	<ul>
	<li>Question:</li>
	
    LSC 0.2.1 runs fine on a small set of data, but when it runs on the whole short read set (49 GB) I get an read error in awk:
	<pre>(awk: read error (Bad address)	</pre>
	At first try it reached 30 GB but this error popped up earlier and earlier with each run. This error also appears when I just run awk from the commandline on this set. 
	</ul>
	<ul>
	<li>Answer:</li>
	The user may have used mawk that is a very fast AWK implementation by Mike Brennan based on a byte code interpreter. You will install GNU awk to run LSC.
	</ul>	
  </div>
  
 <div id="sidebar">
    <h2>Site Map</h2>
    <ul>
      <li><a href="LSC.html">Home</a></li>
      <li><a href="LSC_howitworks.html">How it works?</a></li>
      <li><a href="LSC_download.html">Downloads</a></li>
      <li><a href="LSC_contact.html">Contact</a></li>
      <li><a href="LSC_requirements.html">Requirements</a></li>
      <li><a href="https://github.com/jason-weirather/LSC/">Code</></li>
    </ul>
	 
<h2>Useful links</h2>
    <ul>
      <li><a href="http://www.augroup.org">Au Lab</a> - Au Lab homepage </li>
    </ul>
    <ul>
      <li><a href="http://www.stanford.edu/group/wonglab/SpliceMap/">SpliceMap</a> - Novel splice junction discovery</li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/LSC/LSC">LSC</a> - Error correction of PacBio long reads </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP/IDP">IDP</a> - Isoform Detection and Prediction  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP-ASE/IDP-ASE">IDP-ASE</a> - Determine haplotypes and quantify allele specific expression  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP-fusion/IDP-fusion">IDP-Fusion</a> - Isoform Detection and Prediction across fusion genes  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP/hESC.html">hESC transcriptome</a> - hESC transcriptome by IDP </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/AlignQC/alignqc">AlignQC</a> - Check an alignment, assess error patterns, and make reports </li>
    </ul>
   

<ul>
      <li><a href="http://www.augroup.org/IDP-denovo.20160829/idp-denovo">IDP-denovo</a> - <i>de novo</i> transcriptome assembly and annotation</li>
    </ul>


<ul>
      <li><a href="http://www.augroup.org/GenoClone/GenoClone/GenoClone">GenoClone</a> - Reveal tumor heterogeneity</li>
    </ul>


 <ul>
      <li><a href="https://github.com/jason-weirather/Au-public">Au lab scripts</a> - Public repository of Au lab scripts and utilities </li>
    </ul>


    <br><ul><center><img src="images/seal.jpg" width="75%" hight="75%"><center></ul>
  </div>
<div id="footer">
    <ul>
      <li>&copy; <a href=" http://www.augroup.org">Kin Fai Au</a>. All Rights Reserved.</li>
      <li><a href="https://www.osu.edu/">The Ohio State University</a>, OH 43210.</li>
    </ul>
</div>


<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-23879262-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();


<!-- end #content -->
</body>

</html>
