<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>LSC -  error correction tool for long reads</title>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<link rel="stylesheet" type="text/css" href="styles/flex_textfont_sans.css" media="all" />
<!-- SANDSTONE AND TILE TEMPLATE (FLEXIBLE-WIDTH, SANS-SERIF TEXT FONT VERSION) PROVIDED BY STANFORD UNIVERSITY INFORMATION TECHNOLOGY SERVICES FOR NON-DEPARTMENTAL OR UNOFFICIAL USE BY STANFORD FACULTY, STAFF AND STUDENTS. DESIGNED IN 2009 BY DAVE REAM, IT SERVICES. FOR POLICIES AND GUIDELINES ON ACCESSIBILITY, COPYRIGHT, AND ON THE USE OF STANFORD'S NAME AND TRADEMARKS, INCLUDING THE UNIVERSITY SEAL, SEE http://soap.stanford.edu, http://www.stanford.edu/group/identity, and http://adminguide.stanford.edu --> 
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-50750315-1', 'uiowa.edu');
  ga('send', 'pageview');
</script>

  </head>
<body>

<!--BEGIN TOP RED BAR -->
<div id="top_navbar">
  <!-- OPTIONAL TOP NAVIGATION OR TEXT AREA INSIDE RED BAR; BAR EXPANDS HEIGHT AUTOMATICALLY TO ACCOMMODATE CONTENT -->
   <ul>
    <li>IDP-fusion IS DEVELOPED BY <a href="http://www.augroup.org/">KIN FAI AU</a> with help from collaborators</li>
  </ul>
  <!-- END OF OPTIONAL TOP RED BAR CONTENT -->
</div>
<!-- END TOP RED BAR -->

<div id="content_container">
  <!-- PAGE TITLE AREA AND BOX -->
  <div id="page_title_area">
    <!-- STANDARD PAGE TITLE; NO DECORATIVE GRAPHIC -->
    <!-- <div id="page_title_leftcontent"> -->
    <div id="page_title_leftcontent">
      <h1>LSC</h1>
      <ul>
        <li>LSC is a long read error correction tool. <br>
        It offers fast correction with high sensitivity <br>
        and good accuracy. <br>

      </ul>
    </div>
    
    <div id="page_title_rightcontent">
	<ul>


	    <li>&nbsp;</li>

		<li><img src="images/LSC.png" width="300" >&nbsp;&nbsp;&nbsp;&nbsp;</li>
		
	</ul>
    </div>

<p class="page_title_area_lower">Latest News:  Major update version 2.0 ... <a href="LSC.html#news">read more</a></p>
    
    
  </div>

 
  
  <!-- END PAGE TITLE AREA AND BOX -->
  <div id="content">
    <h2>.cfg file format</h2>
    In order to run LSC. Now, simply edit the run.cfg file and type
    </p>
    <blockquote>
    runLSC run.cfg
    </blockquote>
    <p>
    and that's it! An example run.cfg file is included with every LSC download and decriptions
    of each field can be found in the comments within the file. Detailed descriptions of each field
    is also below. 
    </p>
    
    <h3><img src="images/dna.png"> .cfg syntax</h3>
    There are only three types of syntax in a .cfg file. 
    <dl>
      <dt>Comment</dt>
      <dd>Any line begining with <b>#</b> is a comment. # must be the first character on the line.  </dd>
      <dt>Single Value</dt>
      <dd>A single value written in the following format
      <blockquote>
        <i>tag</i> = <i>value</i>
      </blockquote>
      For example,
      <blockquote>
        genome_dir = ../../genome/hg18
      </blockquote>
      </dd>
      <dt>Multiple Values</dt>
      <dd>Multiple values are grouped by &gt; and &lt; in the following format
      <blockquote>
        &gt; <i>tag</i><br>
        <i>value_1</i><br>
        <i>value_2</i><br>
        ...<br>
        &lt;
      </blockquote>
      </dd>
    </dl>
    
    <h3><img src="images/dna.png"> Example .cfg file</h3>
    <p>
    The following is an example .cfg file. It is included in every LSC download
    or you can download it here <a href="files/run.zip"><img src="images/download.png"> run.cfg</a> (right-click and save as ...). The values
    which are optional are noted in the comments and you may comment them out. 
    </p>

    <pre>
##
###################################################
#
# This cofiguration file contains all settings for a run
# of LScorr.
#
# lines begining with '#' are comments
# lists begin with '> tag' and end with '<' on separate lines
#
###################################################
##

#########################
## Required Settings
##

##
# python path 
# (single value)

python_path = /usr/bin/python

##
# Run mode 
# (single value)
# 0: end-to-end
# 1: alignment stage (generating compressed SRs to LRs alignments)
# 2: correcting stage (assuming stage 1 is already done )

mode = 0

##
# Long reads file
# (single value)

LR_pathfilename = data_path/LR.fa

##
# Long reads file type
# (single value:  fa or fq)

LR_filetype = fa


##
# Short reads file
# (single value)

SR_pathfilename = data_path/SR.fa

##
# Short reads file type
# (single value:  fa or fq or cps)
# If you have run LSC on the same SR data before, you can find the compressed SR data in temp folder (SR.fa.cps and SR.fa.idx files). 
# You can point the SR_pathfilename to SR.fa.cps file (the same folderpath should also include SR.fa.idx file) 
# In this case generating compressed short reads would be skipped
# (single value)

SR_filetype = fa


##
# Is this nonredundant SR data set? (Y or N)
# If you have run LSC on the same SR data before, you could find it in temp folder. Its name is "SR_uniq.fa".
# You can use this "SR_uniq.fa" as the short reads file and set this option to "Y"
# (single value)

I_nonredundant = N

## 
# Short-reads coverage depth (SCD)
# Generates LR-SR alignemnt file with expected SR coverage depth of SCD value.
# Note: SCD filter would be applied to LR segments with SR coverage of more than SCD value. 
# -1: All alignemnt results are used in correction step (no filtration).
# positive integer: Filters SR-LR alignment results to have expected SR coverage depth of SCD. 
# (positive integer or -1)

SCD = 20

##
# Number of threading for short reads alignment to long reads
# (single value)

Nthread1 = 10

##
# Number of threading for corrections
# (single value)

Nthread2 = 10

##
# Max memory usage for unix sort command (-S option) per thread depending on your system memory limit
# Note: This setting is per thread and number of threads is set through Nthread1 and Nthread2 parameters
# -1: no-setting (default sort option) 
# example: 4G , 100M , ...

sort_max_mem = -1

#########################

##
# Temp folder
# (single value)

temp_foldername = temp

##
# Output folder
# (single value)

output_foldername = output


##
# Remove PacBio tails sub reads? (Y or N)
# The names of PacBio long reads must be in the format of the following example: ">m111006_202713_42141_c100202382555500000315044810141104_s1_p0/16/3441_3479"
# The last two numbers (3441 and 3479 in this example) are the positions of the sub reads. 
# (single value)

I_RemoveBothTails = Y

##
# Min. number of non'N' after compressing 
# (single value)

MinNumberofNonN = 39

##
# Max 'N' are allowed after compressing
# (single value)

MaxN = 1

##
# Remove intermediate  files at the end of LSC run (for instance:  aligner sam output, LR-SR mapping files, ...)
# (0: No, 1: Yes )

clean_up = 1


#########################

##
# Aligner could be set to novoalign, bwa or bowtie2

aligner = bowtie2

# Aligner command options   
# Note: Do not specify number of threads in the command options, it is set through Nthread1

novoalign_options =  -r All -F FA  -n 300 -o sam -o 

bwa_options =  -n 0.08 -o 20 -e 3 -d 0 -i 0 -M 1 -O 0 -E  1 -N 

bowtie2_options = --end-to-end -a -f -L 15 --mp 1,1 --np 1 --rdg 0,1 --rfg 0,1 --score-min L,0,-0.08 --no-unal

razers3_options = -i 92 -mr 0 -of sam 





    </pre>
   
  </div>
  
<div id="sidebar">
    <h2>Site Map</h2>
    <ul>
      <li><a href="LSC.html">Home</a></li>
      <li><a href="LSC_howitworks.html">How it works?</a></li>
      <li><a href="LSC_download.html">Downloads</a></li>
      <li><a href="LSC_contact.html">Contact</a></li>
      <li><a href="LSC_requirements.html">Requirements</a></li>
      <li><a href="https://github.com/jason-weirather/LSC/">Code</></li>
  
    </ul>
 <h3>Help</h3>
    <ul>
      <li><a href="LSC_tutorial.html">Tutorial</a></li>
      <li><a href="LSC_manual.html">Manual</a></li>
      <li><a href="http://www.augroup.org/utility_script.html">Utilities</a></li>
      <li><a href="LSC_faq.html">FAQ</a></li>
    </ul>
    <h2>Download</h2>
    <h3>Current release</h3>
    <ul>
      <li><a href="LSC_download.html"><img src="images/download.png"> LSC 2.0</a> (<a href="LSC_download.html#2.0">notes</a>)</li>
    </ul>

<h2>Useful links</h2>
    <ul>
      <li><a href="http://www.augroup.org">Au Lab</a> - Au Lab homepage </li>
    </ul>
    <ul>
      <li><a href="http://www.stanford.edu/group/wonglab/SpliceMap/">SpliceMap</a> - Novel splice junction discovery</li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/LSC/LSC">LSC</a> - Error correction of PacBio long reads </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP/IDP">IDP</a> - Isoform Detection and Prediction  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP-ASE/IDP-ASE">IDP-ASE</a> - Determine haplotypes and quantify allele specific expression  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP-fusion/IDP-fusion">IDP-Fusion</a> - Isoform Detection and Prediction across fusion genes  </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/IDP/hESC.html">hESC transcriptome</a> - hESC transcriptome by IDP </li>
    </ul>
    <ul>
      <li><a href="http://www.augroup.org/AlignQC/alignqc">AlignQC</a> - Check an alignment, assess error patterns, and make reports </li>
    </ul>
   

<ul>
      <li><a href="http://www.augroup.org/IDP-denovo.20160829/idp-denovo">IDP-denovo</a> - <i>de novo</i> transcriptome assembly and annotation</li>
    </ul>


<ul>
      <li><a href="http://www.augroup.org/GenoClone/GenoClone/GenoClone">GenoClone</a> - Reveal tumor heterogeneity</li>
    </ul>


 <ul>
      <li><a href="https://github.com/jason-weirather/Au-public">Au lab scripts</a> - Public repository of Au lab scripts and utilities </li>
    </ul>


    <br><ul><center><img src="images/seal.jpg" width="75%" hight="75%"><center></ul>
  </div>
<div id="footer">
    <ul>
      <li>&copy; <a href=" http://www.augroup.org">Kin Fai Au</a>. All Rights Reserved.</li>
      <li><a href="https://www.osu.edu/">The Ohio State University</a>, OH 43210.</li>
    </ul>
</div>
  

<!-- end #content -->
</body>


<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-23879262-1']);
  _gaq.push(['_trackPageview']);
  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>


</html>
    
